cmd/auth-rest/startcmd/start.go

```diff
@@ -245,7 +245,7 @@ func startAuthService(parameters *authRestParameters, srv server) error {
 	}
 
 	// TODO #34 add the handlers from this controller to the router so that the endpoints are reachable
-	_, err = restapi.New(&operation.Config{Provider: provider})
+	_, err = restapi.New(&operation.Config{TransientProvider: memstore.NewProvider(), PersistentProvider: provider})
```

Comment:
I think calling these `TransientStoreProvider` and `StoreProvider` is more relevant. Also Persistent is redundant when mentioning Store.

Reply:
Done